packages/bundle/src/markdown/markdownItPlugins/respectCRLF.ts

```diff
@@ -0,0 +1,7 @@
+// TODO: We should find some other ways to do this.
+const pre = (markdown: string): string =>
+  // IE11 does not support "u" flag and Babel could not remove it. We intentionally omitting the "u" flag here.
+  // eslint-disable-next-line require-unicode-regexp
+  markdown.replace(/\n\r|\r\n/g, carriageReturn => (carriageReturn === '\n\r' ? '\r\n' : '\n\r'));
```

Comment:
This was one of my first PRs in Web Chat <3

Reply:
Ha, I am still puzzled when I was reading this. It looks like flipping CRLF <-> LRCF, rather than unifying them.